File-Formats/installation-and-upgrade/System-Requirements.md

```diff
@@ -55,5 +49,7 @@ Syncfusion File format libraries supports the following .NET Framework and .NET
 * .NET Framework 4.0
 * .NET Framework 4.5
 * .NET Framework 4.6
-* .NET Standard 2.0 (Can be used in .NET Core 2.1/3.1, .NET 5.0 application)
+* .NET Standard 2.0 (Can be used in .NET Core 2.1/3.1, .NET 5.0, .NET 6.0 application)
+
+You can use our file format libraries in any type of .NET application such as WinForms, WPF, Web services, ASP.NET Web Forms, ASP.NET MVC, ASP.NET Core, Blazor, UWP, Xamarin, WinUI and .NET MAUI applications. 
```

Comment:
You can use our file format libraries in any type of .NET application, like WinForms, WPF, Web services, ASP.NET Web Forms, ASP.NET MVC, ASP.NET Core, Blazor, UWP, Xamarin, WinUI and .NET MAUI applications.

Reply:
Addressed